.github/workflows/pre-commit.yml

```diff
@@ -12,4 +12,25 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: 3.8
-    - uses: pre-commit/action@v2.0.0
\ No newline at end of file
+    - uses: pre-commit/action@v2.0.0
+  
+  generate_json:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - run: git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
```

Comment:
I don't think this is correct to be here, since we want the branch from the user who create the PR.